src/components/AppLayout/DetailedContent.tsx

```diff
@@ -20,19 +21,22 @@ const getLayoutAreas = (useSingleColumn: boolean, isTablet: boolean) => {
 export const DetailedContent: React.FC<DetailedContentProps> = ({
   children,
   useSingleColumn = false,
+  constHeight = false,
+  ...props
 }) => {
   const isTablet = useMediaQuery(tabletMediaQuery);
 
   return (
     <Box
       as="div"
       display="grid"
-      height="100%"
+      height={constHeight ? "100vh" : "100%"}
       margin="auto"
       __maxWidth={contentMaxWidth}
       __gridTemplateColumns={useSingleColumn ? "1fr" : "9fr 4fr"}
       __gridTemplateRows="auto 1fr"
       __gridTemplateAreas={getLayoutAreas(useSingleColumn, isTablet)}
+      {...props}
```

Comment:
**question**: do you need to spread the props?

Reply:
actually not - this is leftover code, good catch